### DISCUSSION TITLE
I'd like to add to the list someone else's repository on GitHub.

### DISCUSSION THREAD

**Discussion by curung** [community/community#60115] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![Screenshot_20230705_173217_Chrome](https://github.com/community/community/assets/138360467/afb20d79-f2a5-4156-ae02-6d6c34be0eaa)
I would like to add like this.

**Comment by GhostOf0days** [comment]
Those are repositories you own or have contributed to in some way (pull requests, issues, a comment on an issue, etc.). Contribute in some way to that repo, and it will appear.

**Reply by GhostOf0days** [reply]
I suggest creating an issue on the repo for something that could be improved.
